### DISCUSSION TITLE
Analog Clock

### DISCUSSION THREAD

**Discussion by bheiland** [FormerLurker/Octolapse#908] [Ideas]
I have been working on adding an analog clock as an option to output.  I may need some help integrating this option.  So far it is hardcoded always enabled.  I'd like to be able to turn it on and off, add parameters for placement on screen, and additional parameters for dimension.  The hour hand is white and barely visible in this image because of the background.  Color of the hands would be another nice option to control.

Thoughts?  I prefer to see the analog movement of hands rather than the digital clock.
Should I continue working on this?

<img width="755" alt="Screenshot 2023-05-13 at 3 05 38 PM" src="https://github.com/FormerLurker/Octolapse/assets/9448014/0bc8e4cd-578d-4379-8c31-8c1ee0ed0b10">

**Comment by FormerLurker** [comment]
Neat!  I saw another thread a long while ago with something similar.

What about adding it as a plugin, and using an event or callback to
generate an image?  Then octolapse could suck it in and merge it as a
layer.  A transparent png or an svg might work.  Thoughts?

**Reply by bheiland** [reply]
Maybe the previous thread was my thread.  I changed the render.py code to get this working.  I lost some of my previous work after an update and decided to revisit this again.  I found a backup of my original changes and decided to work them into the latest render.py that I am running.  I wish I could turn it on and off like the standard decimal clock.  If I can make the changes work I could see if you would like to integrate the feature.

**Reply by bheiland** [reply]
Here is a link to a video, changed face from transparent to white.
https://youtu.be/SQHpUmwzY0g

**Reply by philipp** [reply]
Very cool idea!
Some quick feedback based on watching your video: the second hand is wildly distracting and not useful, at least when it's a timelapse, would be good to be able to turn it off (or default it to off)

**Reply by bheiland** [reply]
It would be easy to turn off second hand.  I agree it is distracting.
